src/xcode/ENA/ENA/Source/Scenes/ContactDiary/Model/ContactPersonEncounter.swift

```diff
@@ -12,18 +12,51 @@ struct ContactPersonEncounter: Equatable {
 		case none
 		case lessThan15Minutes
 		case moreThan15Minutes
+
+		var germanDescription: String {
+			switch self {
+			case .none:
+				return ""
+			case .lessThan15Minutes:
+				return "< 15 Minuten"
+			case .moreThan15Minutes:
+				return "> 15 Minuten"
```

Comment:
Wouldn't it be better to outsource all German strings to a kind of NoLocalizedAppStrings.swift?
Something like: ```static let lessThen15Minues = "< 15 Minuten"```

Reply:
Was thinking about that, but I hope, that there will be a translated export in the future and we move this strings to the current strings file.